cli/js/timers.ts

```diff
@@ -69,36 +50,37 @@ function schedule(timer: Timer, now: number): void {
   // Append the newly scheduled timer to the list and mark it as scheduled.
   dueNode!.timers.push(timer);
   timer.scheduled = true;
-  // If the new timer is scheduled to fire before any timer that existed before,
-  // update the global timeout to reflect this.
-  if (globalTimeoutDue === null || globalTimeoutDue > timer.due) {
-    setOrClearGlobalTimeout(timer.due, now);
+
+  // If tree node has no rid that means that timeout wasn't yet scheduled in Rust - do it now
+  if (!dueNode.rid) {
+    dueNode.rid = sendSync(dispatch.OP_SET_TIMEOUT, { timeout });
+    pollTimeout(dueNode.rid, dueNode.due);
```

Comment:
It seems that `setTimeout()` would now require two ops to be called... Perhaps not such a big deal, but it is less than currently (which is usually 1 op, but sometimes 0 ops if another timer is already pending)

Reply:
It does, but only if there is no other timeout scheduled for that instant